.gitlab/repository.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "Jollybot",
+  "url": "https://gitlab.com/jrbeverly/Jollybot",
```

Use lowercase name for this.

```suggestion
  "url": "https://gitlab.com/jrbeverly/jollybot",
```